python/activator/middleware_interface.py

```diff
@@ -370,8 +402,12 @@ def run_pipeline(self, visit: Visit, snaps: set) -> None:
         exposures = set(self.butler.registry.queryDataIds(["exposure"]))
         self.define_visits.run(exposures)
 
+        # TODO: temporary workaround for uploader and image header not agreeing on what the exposure ID is
+        # This only works because we can't support multiple visits anyway.
```

Comment:
"this only works..." -> "We use the full exposures list here because we don't support multiple visits anyway."